client/src/pages/Feed.js

```diff
@@ -327,7 +327,7 @@ const Feed = (props) => {
     sessionStorage.removeItem("likePost");
 
     if (isAuthenticated) {
-      const endPoint = `/api/posts/${postId}/likes/${user && user.id}`;
+      const endPoint = `/api/posts/${postId}/likes/${user && user._id}`;
```

Comment:
Oh I thought you were going to change backend too from our conversation on slack. 

This will work after sign-up but now it doesn't work later on because of: https://github.com/FightPandemics/FightPandemics/blob/a09c8b314a8d818b1c1964a524564f423979ddd6/backend/lib/endpoints/users.js#L31. 

Let's take more time to do overall full stack update (to use `_id` only) with a resolution to #923 

For now to fix the bug do something similar to https://github.com/FightPandemics/FightPandemics/blob/a09c8b314a8d818b1c1964a524564f423979ddd6/client/src/components/Header.js#L123

```suggestion
      const endPoint = `/api/posts/${postId}/likes/${user?.id || user?._id}`;
```

Reply:
Will do! I'll fix the bug for now, but I too would like to fix this issue in a more permanent way.

Reply:
Agreed - assign yourself to the other issue and start working on that one after this is merged.